drivers/sm_pwm_01c/sm_pwm_01c_saul.c

```diff
@@ -55,12 +55,10 @@ const saul_driver_t sm_pwm_01c_saul_driver_mc_pm_10 = {
     .read = read_mc_pm_10,
     .write = saul_notsup,
     .type = SAUL_SENSE_PM,
-    .subtype = SAUL_SENSE_PM_10,
```

Comment:
sneaked in upon rebasing?

Reply:
It is intentional because the `SAUL_SENSE_PM_10/2p5` defines don't exist in the repo:

```
$ git grep SAUL_SENSE_PM_10
drivers/sm_pwm_01c/sm_pwm_01c_saul.c:    .subtype = SAUL_SENSE_PM_10,
$ git grep SAUL_SENSE_PM_2p5
drivers/sm_pwm_01c/sm_pwm_01c_saul.c:    .subtype = SAUL_SENSE_PM_2p5,
```

Same for the subtype attribute: it doesn't exist in the SAUL struct.

In fact, this file was never built by Murdock because the saul.c file was skipped and there's no default configuration that trigger a build of this file, which is fixed by this PR.